delira/training/metrics.py

```diff
@@ -1,106 +1,37 @@
-from sklearn.metrics import roc_auc_score
-from sklearn.metrics import accuracy_score
+from sklearn.metrics import accuracy_score, balanced_accuracy_score, f1_score, \
+    fbeta_score, hamming_loss, jaccard_similarity_score, log_loss, \
+    matthews_corrcoef, precision_score, recall_score, zero_one_loss
 from ..utils.decorators import make_deprecated
 
 import trixi
+import numpy as np
 
 from delira import get_backends
 
-if "TORCH" in get_backends():
-    import torch
 
-    from .train_utils import pytorch_tensor_to_numpy, float_to_pytorch_tensor 
-    @make_deprecated(trixi)
-    class AurocMetricPyTorch(torch.nn.Module):
-        """
-        Metric to Calculate AuROC
+class SklearnClassificationMetric(object):
+    def __init__(self, score_fn):
+        self._score_fn = score_fn
 
-        .. deprecated:: 0.1
-            :class:`AurocMetricPyTorch` will be removed in next release and is
-            deprecated in favor of ``trixi.logging`` Modules
+    def __call__(self, y_true, y_pred, **kwargs):
+        arrays = {"y_true": y_true, "y_pred": y_pred}
 
-        .. warning::
-            :class:`AurocMetricPyTorch` will be removed in next release
+        for k, v in arrays.items():
+            arrays[k] = v.reshape(v.shape[0], -1).astype(np.uint8)
 
-        """
-        def __init__(self):
-            super().__init__()
+        return self._score_fn(**arrays, **kwargs)
 
```

Comment:
The AUC score would be a nice addtional feature. If needed, I can provide an implementation later today or tomorrow.

Reply:
Feel free to do so.

Reply:
@mibaumgartner please resolve if done or no longer planned

Reply:
Sorry, I didn't get any notification. I already sketched a first version, gonna test it during the next days (maybe even today) and then create a pull request.